builder/azure/arm/builder_acc_test.go

```diff
@@ -10,6 +10,8 @@ package arm
 // * ARM_CLIENT_SECRET
 // * ARM_SUBSCRIPTION_ID
 // * ARM_STORAGE_ACCOUNT
+// * AZURE_SSH_PASS - In our Linux SIG test we build two specialized images, this requires setting a shared password for both builds
```

Comment:
I recommend adding a description to explain how this password is used.
```suggestion
// * ARM_SSH_PASS - In our Linux SIG test we build two specialized images (parent, child), which requires setting a shared password that can be used in both builds. The password is needed for .......
```

Reply:
We actually didn't need it 😅, I've removed it